packages/settingeditor/src/pluginlist.tsx

```diff
@@ -279,39 +365,54 @@ export class PluginList extends ReactWidget {
         key={id}
         title={itemTitle}
       >
-        {id === this.selection || this._errors[id] ? (
-          <div className="jp-SelectedIndicator" />
-        ) : null}
-        <LabIcon.resolveReact
-          icon={icon || (iconClass ? undefined : settingsIcon)}
-          iconClass={classes(iconClass, 'jp-Icon')}
-          title={iconTitle}
-          tag="span"
-          stylesheet="settingsEditor"
-        />
-        <span>{title}</span>
+        <div>
```

Comment:
Oh ok I'll remove `width: -webkit-fill-available`, not sure why that was in either. 

Are there accessibility considerations for using `div` instead of the `button` tag? Should we add an aria role of some kind?

Reply:
Yes, I think that adding an aria role would be appropriate when replacing the `button` with a `div`. I think that [`tab` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) is appropriate here.